filters/diacritic_filter/filter.py

```diff
@@ -0,0 +1,19 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import unicodedata
+
+class DiacriticFilter(SentenceOperation):
+
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
```

Comment:
Please add Docstrings for your filter class.

Reply:
Thanks, added the docstring now.